webpack/assets/javascripts/react_app/common/sessionStorage.js

```diff
@@ -0,0 +1,23 @@
+if (!window.sessionStorage) {
```

Comment:
it's not a part of test, this is a service file for sessionStorage encapsulation.